### DISCUSSION TITLE
npm ERR! code EINTEGRITY

### DISCUSSION THREAD

**Discussion by rono-koushiq** [vercel/community#4705] [Help] [answered]
### Summary

I have deployed a few apps on vercel before but getting this error for the first time. Need Help

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/69637419/02c9df0f-9519-4a2e-ad1a-c26674572120)

**Comment by kitavidavis** [comment]
@Rono-Koushique did you find solution to this? I am experiencing the same issue.

**Reply by rono-koushiq** [reply]
no. unfortunately the project i was working on got cancelled.

**Reply by kitavidavis** [reply]
Okay.

**Comment by kitavidavis** [comment, marked as answer]
Just in case someone might face this issue someday, the solution is to delete the package-lock.json file and re-deploy. It worked for me.
